encrypted-media-respec.html

```diff
@@ -332,7 +332,7 @@ <h2>Definitions</h2>
 
           <p>
           The format of the initialization data depends upon the type of container, and containers MAY support more than one format
-          of initialization data. The <dfn class="export" id="initialization-data-type">Initialization Data Type</dfn> is a string that indicates the
```

Comment:
Can this ID be dropped? I suspect ReSpec will create a different ID (`dfn-initialization-data-type`) and the spec contains references to the term that use the [linking system in `encrypted-media.js`](https://github.com/w3c/encrypted-media/blob/2e4580cabc4bdd150547db7c362a6652d0689a71/encrypted-media.js#L172), which expects the ID to exist.

Reply:
Ah, let me restore this for now, then.

Reply:
Done